File: HackerEarth/Game of multiples.py
#  Game of multiples

# https://www.hackerearth.com/problem/algorithm/multiples/

t=int(raw_input())
for i in range(t):
	n=int(raw_input())
	n=n-1
	th=int(n/3)
	fi=int(n/5)
	s=int(0)
	fif=int(n/15)
	s=(th*(6+(th-1)*3))/2
	s=s+(fi*(10+(fi-1)*5))/2
	s=s-(fif*(30+(fif-1)*15))/2
	print s

File: HackerEarth/Game of multiples.java
// Game of multiples

// https://www.hackerearth.com/problem/algorithm/multiples/

import java.util.*;
class TestClass {
    public static void main(String args[] ) throws Exception {

        Scanner ss=new Scanner(System.in);
        long t=ss.nextLong();
        while(t!=0)
        {
        	long n=ss.nextLong();
        	n=n-1;
        	long th=n/3;
        	long fi=n/5;
        	long fif=n/15;
        	long s=0;
    s=(th*(6+(th-1)*3))/2;
	s=s+(fi*(10+(fi-1)*5))/2;
	s=s-(fif*(30+(fif-1)*15))/2;
        System.out.println(s);
        t--;
        }
    }
}